fe/fe-core/src/main/java/com/starrocks/credential/aws/AWSCloudConfiguration.java

```diff
@@ -98,11 +133,17 @@ public CloudType getCloudType() {
 
     @Override
     public FileStoreInfo toFileStoreInfo() {
-        return awsCloudCredential.toFileStoreInfo();
+        FileStoreInfo.Builder builder = awsCloudCredential.toFileStoreInfo().toBuilder();
+        // update the FileStoreInfo with enablePartitionedPrefix and numOfPartitionedPrefix
+        builder.getS3FsInfoBuilder()
+                .setPartitionedPrefixEnabled(enablePartitionedPrefix)
+                .setNumPartitionedPrefix(numOfPartitionedPrefix);
+        return builder.build();
     }
 
     @Override
     public String toConfString() {
+        // TODO: add enable_partitioned_prefix, num_partitioned_prefix output
```

Comment:
why todo, we will use `toConfString` as map's key.

Reply:
these two properties are specific for storage volume, won't be used for external table property configuration. So add todo here.